src/constants/form.js

```diff
@@ -0,0 +1,3 @@
+export const INPUT_TYPES = ["text", "email", "password", "number", "submit", "checkbox", "date", "file"];
```

Comment:
Should this be an array or object ? 
Also I dont see a date field or a file field or a number field

Reply:
Lets Check for date. Also change typeValueMapping for files

Reply:
done